pkg/cmd/server/api/validation/master.go

```diff
@@ -79,6 +80,8 @@ func ValidateMasterConfig(config *api.MasterConfig) fielderrors.ValidationErrorL
 		allErrs = append(allErrs, ValidateOAuthConfig(config.OAuthConfig).Prefix("oauthConfig")...)
 	}
 
+	allErrs = append(allErrs, ValidateProjectNodeSelector(config.ProjectNodeSelector)...)
```

Comment:
I am a little confused by this option.  Are we saying that there is a default node selector that gets applied on all project creation requests unless the user provides a different value?